reliability-engineering/terraform/modules/concourse-monitoring/grafana.tf

```diff
@@ -156,16 +156,30 @@ resource "grafana_data_source" "cloudwatch" {
 }
 
 locals {
-  grafana_dashboards = [
+  grafana_dashboards = toset([
     "alerts",
     "concourse",
     "concourse-internal",
+    "concourse-slo",
     "grafana-metrics",
     "metrics-by-team"
-  ]
+  ])
+}
+
+data "template_file" "grafana_dashboards" {
```

Comment:
non-blocking: new code should really use the `templatefile` function, this data source is the 0.11-and-earlier way of doing things.

Reply:
turns out this was blocking after all 😄  #186